### PR TITLE
Awesome Books: Project 1

### PR DESCRIPTION
**Pull request by V0pe**
### Awesome Books: Plan HTML and CSS

In this project, the following was carried out,

- [ ] Add the markup for the landing page.
- [ ] Create the new book class and append it to the book list.
- [ ] Add the local storage and retrieve the data from the localstorage. 
- [ ] Append the new and updated data to the book list section.
- [ ] Implement the remove button that removes each item from the book list.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,55 @@
...
+
+Prerequisites
+>- VS code 
+>- Git Bash
+
+Setup
+>- Git clone `git@github.com:rolandfbk/awesome-books.git`
+
+
+## Author
```
Comment (prolajumokeoni) [inline README.md:33]: OPTIONAL 
- [ ] Kindly add social media  links of other collaborators

File: index.js [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+    btn.addEventListener('click', () => {
+      booksrecord = booksrecord.filter((book) => book.id !== Number(btn.id));
+      bookRec.innerHTML = '';
+      booksrecord.forEach((book) => {
+        displayBook(book.title, book.author, book.id);
+      });
+      localStorage.setItem('bookStorage', JSON.stringify(booksrecord));
+    });
+  });
+}
```
Comment (prolajumokeoni) [inline index.js]: - [ ] When I click on the remove button: 
- The right book is not removed from the collection. 
- The right book does not disappear from the page. 
### Kindly fix this as this is part of the  project requirements 👍🏿 
- [ ] Kindly ensure that data is preserved in the browser's memory by using [localStorage](https://github.com/microverseinc/curriculum-html-css/blob/main/local_storage.md) after removing a book.
.

**Review by prolajumokeoni — CHANGES_REQUESTED** [review]
Hi <STUDENT_MENTION>, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
✔️ No linter errors
✔️  Add button works
✔️  Good PR and Description



## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
_Check inline comments_
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @prolajumokeoni.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by EricMbouwe — APPROVED** [review]
### Hi @V0pe, 

# Status: Approved ✔️
Excellent job on this project. Keep it up 👏
Your project is complete! There is nothing else to say other than... it's time to merge it 💪
Congratulations! 🎉

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**